ludwig/encoders/generic_encoders.py

```diff
@@ -26,7 +26,7 @@
 logger = logging.getLogger(__name__)
 
 
-@register_encoder("passthrough", [CATEGORY, NUMBER, VECTOR])
+@register_encoder("passthrough", [NUMBER, VECTOR])
```

Comment:
I think I'm missing some context; what is the use case for this?

Categorical variables aren't continuous so I don't know why we would want a passthrough encoder for them.

Reply:
I don't think there's a new use case for this. We already support passthrough encoder for categorical features, but specifying it as an encoder in the config leads to an error.

Reply:
The passthrough encoder for category features is used by the gbm model type. CC: @jppgks

Reply:
Aha, that explains it.  Makes sense for decision trees.